isis/src/osirisrex/objs/OsirisRexOcamsCamera/OsirisRexOcamsCamera.cpp

```diff
@@ -50,7 +50,7 @@ namespace Isis {
       m_instrumentNameLong = "Sampling Camera";
       m_instrumentNameShort = "SamCam";
     } // IK values for polycam: -64360 (general) and -64616 to -64500 (focus specific)
-    else if (frameCode == -64360) {
+    else if ((frameCode == -64360) || (frameCode >= -64616 && frameCode <= -64500)) {
```

Comment:
That's a large range of numbers, what is the significance of the IK codes. 

I'm sure it's correct just curious.

Reply:
The IK codes are different focus positions for the sensor.